joplin/PKGBUILD

```diff
@@ -49,11 +48,15 @@ _get_yarn_bin() {
 prepare() {
     local cache=$(_get_cache)
     local yarn_bin=$(_get_yarn_bin)
+    local package_json="${srcdir}/joplin-${pkgver}/package.json"
     msg2 "Yarn cache directory: $cache"
     msg2 "Yarn binary: ${yarn_bin}"
 
     msg2 "Disabling husky (git hooks)"
-    sed -i '/"husky": ".*"/d' "${srcdir}/joplin-${pkgver}/package.json"
+    # This is ugly and unreadable, but it gets rid of "husky" everwhere without breaking package.json
```

Comment:
Since husky is the last listed dependency in `package.json` now, it breaks when the sed line is run.

See: https://github.com/laurent22/joplin/blob/dev/package.json#L111